fetch/src/3_visualize.r

```diff
@@ -0,0 +1,45 @@
+plot_data <- function() {
+  # Create a plot
+  png(file = file.path(project_output_dir, 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
And here, you'll need to specify that the output directory for this phase is `3_visualize/out`